sesman/chansrv/smartcard_pcsc.c

```diff
@@ -120,18 +109,18 @@ get_uds_client_by_id(int uds_client_id)
     struct pcsc_uds_client *uds_client;
     int index;
 
-    LLOGLN(10, ("get_uds_client_by_id:"));
+    LOG_DEVEL(LOG_LEVEL_DEBUG, "get_uds_client_by_id:");
     if (uds_client_id == 0)
     {
-        LLOGLN(10, ("get_uds_client_by_id: uds_client_id zero"));
+        LOG_DEVEL(LOG_LEVEL_ERROR, "get_uds_client_by_id: uds_client_id zero");
```

Comment:
Replace `LOG_DEVEL` with `LOG` in all places where `LOG_LEVEL_ERROR` is used in this file.

Reply:
I'm not sure what a sysadmin will do with the log messages like "ERROR: uds_client_id is zero", since this seems more like a bug than a misconfiguration.
I'm deferring to you that having these messages in the release version's log makes sense.

Reply:
Fair point! It's a poor message, but probably better than nothing at all.